src/learning-header/AuthenticatedUserDropdown.jsx

```diff
@@ -40,18 +48,22 @@ const AuthenticatedUserDropdown = ({ intl, username }) => {
               </Dropdown.Item>
             </>
           )}
+          <span className="dropdown-section-name with-border">{intl.formatMessage(messages.manageInformation)}</span>
+          <Dropdown.Item href={getConfig().ACCOUNT_SETTINGS_URL}>
+            {intl.formatMessage(messages.myAccount)}
+          </Dropdown.Item>
           <Dropdown.Item href={`${getConfig().ACCOUNT_PROFILE_URL}/u/${username}`}>
-            {intl.formatMessage(messages.profile)}
+            {intl.formatMessage(messages.myProfile)}
           </Dropdown.Item>
-          <Dropdown.Item href={getConfig().ACCOUNT_SETTINGS_URL}>
-            {intl.formatMessage(messages.account)}
+          <Dropdown.Item href="https://edx.readthedocs.io/projects/open-edx-learner-guide/en/open-release-quince.master/SFD_dashboard_profile_SectionHead.html">
```

Comment:
Are we sure that it's a good way to set links like this one? Maybe it should be taken from the config?

As we have above:
```
<a className="text-gray-700" href={`${getConfig().SUPPORT_URL}`}>{intl.formatMessage(messages.help)}</a>
```

Reply:
Fixed